Sources/inc/CosmicCORSIKA.hh

```diff
@@ -231,6 +231,7 @@ namespace mu2e {
 
       const float _GeV2MeV = CLHEP::GeV / CLHEP::MeV;
       const float _cm2mm = CLHEP::cm / CLHEP::mm;
+      const float _ns2s = CLHEP::ns / CLHEP::s;
```

Comment:
Please make these:

 static constexpr float

I just checked that the CLHEP definitions are constexpr so this will work. 

In this case it's not very important ( I bet you only ever make on CosmicCORSIKA object per job) - so you are not really wasting memory.   I am just getting it on the radar for when it is important.

Reply:
FIxed, thanks.